src/lib/operators/table_scan/column_is_null_table_scan_impl.hpp

```diff
@@ -27,6 +28,8 @@ class ColumnIsNullTableScanImpl : public AbstractTableScanImpl {
 
  protected:
   void _scan_generic_segment(const BaseSegment& segment, const ChunkID chunk_id, RowIDPosList& matches) const;
+  void _scan_generic_ordered_segment(const BaseSegment& segment, const ChunkID chunk_id, RowIDPosList& matches,
```

Comment:
Nothing I would change on the remaining occurrences:
```
➜  deb git:(sort-in-operators) ✗ grep -r ordered ../src/ | grep -vi unordered | grep -vi reordered
../src//test/logical_query_plan/aggregate_node_test.cpp:    // Columns are ordered as specified in the SELECT list
../src//test/optimizer/strategy/predicate_reordering_rule_test.cpp:     * This is particularly interesting because the PredicateReorderingRule needs to re-attach the ordered chain of
../src//lib/cache/lru_k_cache.hpp:    // Reference history, ordered with the least-recent access at the beginning.
../src//lib/strong_typedef.hpp:  struct D : boost::totally_ordered1<D> {                                                                         \
../src//lib/optimizer/join_ordering/enumerate_ccp.cpp:   * Returns all subsets of `vertex_set`, ordered in a subsets-first order, e.g., for 01101 returns
../src//lib/utils/size_estimation_utils.hpp:  // Since we want an ordered position list (this potentially increases the performance when accessing the segment), we
../src//lib/storage/run_length_segment/run_length_segment_iterable.hpp:   * ordered and modern CPUs are well at prefetching, linear searches are often faster than a complete binary search.
../src//lib/storage/run_length_segment/run_length_segment_iterable.hpp:   * linear search in O(n). More often than not, the chunk offsets will be ordered,
../src//lib/storage/index/abstract_index.hpp:   * Iterating from cbegin() to cend() will result in a position list with ordered values.
../src//lib/storage/index/abstract_index.hpp:   * Iterating from cbegin() to cend() will result in a position list with ordered values.
../src//benchmarklib/benchmark_runner.hpp:  void _benchmark_ordered();
../src//benchmarklib/tpch/tpch_queries.hpp: * Contains all supported TPCH queries. Use ordered map to have queries sorted by query id.
../src//benchmarklib/benchmark_runner.cpp:      _benchmark_ordered();
../src//benchmarklib/benchmark_runner.cpp:void BenchmarkRunner::_benchmark_ordered() {
../src//benchmarklib/benchmark_runner.cpp:    // For ordered benchmarks, report the time that this individual item ran. For shuffled benchmarks, return the
../src//benchmarklib/benchmark_runner.cpp:    // duration of the entire benchmark. This means that items_per_second of ordered and shuffled runs are not
../src//benchmarklib/benchmark_runner.cpp:  // Make TPC-C run in shuffled mode. While it can also run in ordered mode, it would run out of orders to fulfill at
```